docs/index.d.ts

```diff
@@ -0,0 +1,512 @@
+import { IncomingMessage, ServerResponse } from "http";
+import * as opentracing from "opentracing";
+import { SpanOptions } from "opentracing/lib/tracer";
+
+/**
+ * Tracer is the entry-point of the Datadog tracing implementation.
+ */
+export declare interface Tracer extends opentracing.Tracer {
+  startSpan(name: string, options?: SpanOptions): Span;
```

Comment:
Should we add documentation to these methods?

Reply:
This is actually a bug in `typedoc` where it should use the documentation from the parent. I'll try to add our own doc for now and figure out why this is happening later.